ARK Server Manager/Lib/ServerRCON.cs

```diff
@@ -370,13 +370,45 @@ private void HandleCommand(ConsoleCommand command)
                     }
                 }).DoNotWait();
             }
+            else
+            {
+                var arkData = await ArkData.ArkDataContainer.CreateRemoteAsync();
+                arkData.SteamLoaded = true;
+                //await arkData.LoadSteamAsync(Config.Default.SteamAPIKey);
+                TaskUtils.RunOnUIThreadAsync(() =>
+                {
+                    foreach (var playerData in arkData.Players)
+                    {
+                        var playerToUpdate = this.Players.FirstOrDefault(p => p.SteamId == Int64.Parse(playerData.SteamId));
+                        if (playerToUpdate != null)
+                        {
+
+                            playerToUpdate.AvatarImage = new BitmapImage(new Uri(playerData.AvatarUrl));
```

Comment:
well.. not sure if it helps, but as mentioned, for some reason my tribe, avatar and playername are not updated. I have added LogEvent calls before "var arkData = await ArkData.ArkDataContainer.CreateAsync(savedArksPath);" which works fine, but when adding it within the Async bit (after "foreach (var playerData in arkData.Players)") it never seems to log and therefore it looks like it never enters the Async bit, similar to what knightzac19 is experiencing with th Async part... so... any thoughts why it never enters the Async part of the code?